schemas/2015-06-01/Microsoft.Cdn.json

```diff
@@ -0,0 +1,222 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.Cdn.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Cdn",
+    "description": "Microsoft Cdn Resource Types",
+    "resourceDefinitions": {
+        "profiles": {
+            "type": "object",
+            "description": "The CDN profile",
+            "properties": {
+                "type": {
+                    "enum": [ "Microsoft.Cdn/profiles" ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2015-06-01"
+                    ]
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "sku": {
```

Comment:
we require all properties need to support expression now. So you need to use "oneOf" to support "expression", please take other schema as an example.